lib/winrm-fs/core/file_transporter.rb

```diff
@@ -451,6 +451,7 @@ def stream_upload(input_io, dest)
           read_size = ((max_encoded_write - dest.length) / 4) * 3
           chunk = 1
           bytes = 0
+          # Do not freeze this string
           buffer = ''
```

Comment:
Is there a different buffer we could use here? It appears the `input_io.read` method will not accept a stringio there. But I did not spend much time on it.